CHANGES.rst

```diff
@@ -19,6 +19,10 @@ Fixes
 - Fixed an issue that caused the node health to not be displayed in the Cluster 
   View when the node name was too large.
 
+- Updated keywords list so that they are recognised and painted in red.
```

Comment:
Why is this added beneath a already released version?

Reply:
keen eye!